docs/source/user_guide/training.rst

```diff
@@ -76,6 +76,26 @@ while the other parameters are optional. Here is an example (can be found in the
     keep_isolated_atoms: True
     save_cpu: True
 
+It is also possible to fine-tune models using the same type of `Calcjob`.
+In that case some additional parameters must be used:
```

Comment:
It's probably worth saying explicitly which new parameters are required here, otherwise you have to check the difference between this and normal training (and assume only the minimum options have been listed in both cases).

Reply:
isn't the only additional thing the model?
 the parameters described here are all those that are possible to pass to aiida: the config file (which was also for the training and I understood it's more or less the same), the model (that can be given as in the example as an input or in the config, and that is for my understanding the only possible difference inside the config) and the fine-tune keyword to "activate" the fine tuning

Reply:
(and then the code and metadata, which you pass to every calcjob)

Reply:
> isn't the only additional thing the model?

I think so, which was part of the point, as "some additional parameters" implies to me that there are more differences to worry about.